ocf_datapipes/training/pvnet_site.py

```diff
@@ -24,6 +24,8 @@
     NWP_MEANS,
     NWP_STDS,
     RSS_MEAN,
+    RSS_RAW_MAX,
+    RSS_RAW_MIN,
     RSS_STD,
```

Comment:
Sorry to be like this, but shouldn't these be in grouped? min-max together and mean-std together

Reply:
I think `isort` orders them alphabetically

Reply:
Ah! Shame. Sorry to nitpick for nothing :')